lib/pages/settings_list_item.dart

```diff
@@ -190,46 +197,11 @@ class _SettingsListItemState<T> extends State<SettingsListItem<T>>
     _handleExpansion();
     final theme = Theme.of(context);
 
-    final optionWidgetsList = <Widget>[];
-
-    widget.options.forEach(
-      (optionValue, optionDisplay) => optionWidgetsList.add(
-        RadioListTile<T>(
-          value: optionValue,
-          title: Column(
-            crossAxisAlignment: CrossAxisAlignment.start,
-            children: [
-              Text(
-                optionDisplay.title,
-                style: theme.textTheme.bodyText1.copyWith(
-                  color: Theme.of(context).colorScheme.onPrimary,
-                ),
-              ),
-              if (optionDisplay.subtitle != null)
-                Text(
-                  optionDisplay.subtitle,
-                  style: theme.textTheme.bodyText1.copyWith(
-                    fontSize: 12,
-                    color: Theme.of(context)
-                        .colorScheme
-                        .onPrimary
-                        .withOpacity(0.8),
-                  ),
-                ),
-            ],
-          ),
-          groupValue: widget.selectedOption,
-          onChanged: (newOption) => widget.onOptionChanged(newOption),
-          activeColor: Theme.of(context).colorScheme.primary,
-          dense: true,
-        ),
-      ),
-    );
-
     return AnimatedBuilder(
       animation: _controller.view,
       builder: _buildHeaderWithChildren,
       child: Container(
+        constraints: const BoxConstraints(maxHeight: 380),
```

Comment:
So is this the real crux of the fix? Just capping the height?

Reply:
Yes, I believe most of the gains are gotten through this

Reply:
since 380 / 8 = 47.5, and if this is to show exactly 8 items, should it maybe be something like 384? (maybe the height of the item is 48?)

Reply:
Good catch, this makes it fit perfectly